presto-hive/src/main/java/com/facebook/presto/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -205,12 +207,23 @@ else if (config.getPinGlueClientToCurrentRegion()) {
 
     @Override
     public Optional<Table> getTable(String databaseName, String tableName)
+    {
```

Comment:
I'm confused about why there are so many changes for the glue hive metastore.  Why isn't this just updating with an enum?

Reply:
Glue uses `get - update - write` approach for `add_column`, `rename_column`, `delete_column`. It fetches the table object from the metastore, updates the column lists, writes the table object back to the metastore. 

Since table type in the `Table` obejct is no longer string, and non standard table types are represented just as `OTHER`, the `Table` object cannot longer be used as an intermediate object for the `get - update - write`. Instead the Glue's native object has to be used.